packages/client/webapp/src/components/Sidebar.vue

```diff
@@ -0,0 +1,134 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import Button from '@/components/Button.vue';
+import SimulationPlan from '@carbon/icons-vue/es/data-player/32';
+import MachineLearningModel from '@carbon/icons-vue/es/machine-learning-model/32';
+import Dataset from '@carbon/icons-vue/es/table--split/32';
+import ProvenanceGraph from '@carbon/icons-vue/es/flow/32';
+import Search from '@carbon/icons-vue/es/search--locate/32';
+import User from '@carbon/icons-vue/es/user/32';
+
+const props = defineProps<{ sidebarPosition: String }>();
+const emit = defineEmits(['updateSidebarPosition']);
+
+const mode = ref('Simulation Plan');
+const collapsed = ref(false);
```

Comment:
It's a useful convention to start boolean variables with `is` or `has`, so this would be `isCollapsed`.